media/css/mozorg/participation-reporting.scss

```diff
@@ -0,0 +1,33 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+
+.reporting-matrix {
+    width: 100%;
```

Comment:
Could it be worth placing this table in a wrapper with `overflow: scroll` to try and avoid it busting out of the page at small screen sizes? It's by no means a perfect solution, but could be a quick fix rather than working on making the table more responsive.

Reply:
good idea... it's a complicated one-off table that needed special treatment anyway, can't hurt to wrap it in a div.